source/installation-guide/installing-wazuh-agent/wazuh_agent_sources.rst

```diff
@@ -42,6 +42,9 @@ Installing Linux agent
     # cd wazuh-*
     # ./install.sh
 
+  .. note::
```

Comment:
Please add a code block like existing ones, and one command per line.
Consider if this way is aesthetically better.

On the other hand, let the user know that the Makefile is in `src`, for instance, suggest `make -C src clean`.

Reply:
- The cleaning commands include the `src` directory and instructions reference the Makefile.
- The cleaning commands are now under a code block instead of a note.